### PR TITLE
chore(weave): add models on dark icon

### PR DESCRIPTION
**Pull request by mbarrramsey**
Add models icon for display on dark screens for the new onboarding flow, https://www.figma.com/file/febPfitJoMdHN4z3xmSDL3/Community-Edition-V2?node-id=5126%3A74857&mode=dev. 

See usage:
Screenshot 2024-03-28 at 3 13 50 PM

### CODE REVIEW

File: weave-js/src/assets/icons/icon-model-on-dark.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+  <path d="M15.6857 4.02113C15.6857 6.02944 14.0576 7.65749 12.0493 7.65749C10.041 7.65749 8.41295 6.02944 8.41295 4.02113C8.41295 2.01282 10.041 0.384766 12.0493 0.384766C14.0576 0.384766 15.6857 2.01282 15.6857 4.02113Z" fill="#E4E6EB"/>
```
Comment (jwlee64) [inline weave-js/src/assets/icons/icon-model-on-dark.svg:2]: i think we want this fill color to be "currentColor"

Comment (mbarrramsey) [reply weave-js/src/assets/icons/icon-model-on-dark.svg:2]: @jwlee64 done!

Comment (mbarrramsey) [reply weave-js/src/assets/icons/icon-model-on-dark.svg]: @jwlee64 done -- changed the top one back to "none"

**Review by jwlee64 — APPROVED** [review]
nice lgtm

**Pull request merged**